rocAL/rocAL/source/rali_api_data_loaders.cpp

```diff
@@ -301,7 +490,7 @@ raliJpegCaffe2LMDBRecordSource(
         output = context->master_graph->create_loader_output_image(info);
 
         context->master_graph->add_node<ImageLoaderNode>({}, {output})->init(internal_shard_count,
-                                                                             source_path, "",
+                                                                             source_path, "", 0, 0, 0,
```

Comment:
not sure why we need to pass sequence related parameters here. Applies to all below

Reply:
Sequence Reader uses the ImageLoaderNode, and it requires the parameters such as sequence_length, frame_step and frame_stride similar to Video Reader. Hence these 3 parameters has been set to 0 in all the image loader nodes (except for sequence reader).